server/koppsApi.js

```diff
@@ -131,9 +150,22 @@ async function getKoppsCourseRoundTerms(courseCode) {
         yearBeforeCurrentYear(t.term)
     )
 
+    if (activeTerms && activeTerms.length > 0) {
+      for await (const term of termsWithCourseRounds) {
+        const { rounds } = term
```

Comment:
Term might not have rounds, so it requires safe extracting f.e., `const { rounds = [] } = term`

Reply:
Done.